packages/calculon/calculon.0.1/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "1.2"
+authors: ["Armael" "Enjolras" "c-cube"]
+maintainer: "c-cube"
+build: [
+  ["./configure"
+    "--bindir" "%{bin}%"
+    "--%{uri+lambdasoup+cohttp+atdgen:enable}%-web"
+    "--%{re+sequence:enable}%-extras"
+    ]
+  [make "build"]
+]
+install: [
+  [make "install"]
+]
+build-doc: [ make "doc" ]
+build-test: [ make "test" ]
+remove: [
+  ["ocamlfind" "remove" "calculon"]
+]
+depends: [
+  "ocamlfind" {build}
+  "base-bytes"
+  "base-unix"
+  "result"
+  "lwt"
+  "irc-client" { >= "0.4.0" }
+  "tls"
+  "yojson"
+  "containers" { >= "1.0" }
+  "ISO8601"
+  "re"
+  "stringext"
+]
+depopts: [
+  "uri"
+  "cohttp"
+  "atdgen"
+  "lambdasoup"
+  "sequence"
+]
+tags: [ "irc" "bot" "factoids" ]
+homepage: "https://github.com/c-cube/calculon"
+bug-reports: "https://github.com/c-cube/calculon/issues"
+dev-repo: "https://github.com/c-cube/calculon.git"
+
+
```

Comment:
This is [failing on 4.01](https://travis-ci.org/ocaml/opam-repository/jobs/208346492) because it uses module aliases:

```
# File "src/core/Core.mli", line 4, characters 11-12:
# Error: Syntax error
# Command exited with code 2.
```